### PR TITLE
QUICK-FIX Delete expected negative fail due to issue GGRC-1773 was fixed

### PR DESCRIPTION
**Pull request by AleksNeStu**

### CODE REVIEW

**Comment by AleksNeStu** [comment]
@SergeyRabtsau or @dnovikau or @ch-t could you please review this one PR.

File: test/selenium/src/tests/test_snapshots.py [hunk 1]
```diff
@@ -144,19 +143,15 @@ def test_audit_contains_snapshotable_control(
...
-    is_control_openable = controls_ui_service.is_obj_page_exist_via_info_panel(
-        src_obj=audit, obj=expected_control)
     is_control_updateable = (
         controls_ui_service.is_obj_updateble_via_info_panel(
             src_obj=audit, obj=expected_control))
+    is_control_openable = controls_ui_service.is_obj_page_exist_via_info_panel(
+        src_obj=audit, obj=expected_control)
     actual_control = controls_ui_service.get_list_objs_from_info_panels(
         src_obj=audit, objs=expected_control)
+    assert is_control_updateable is True
```
Comment (dnovikau) [inline test/selenium/src/tests/test_snapshots.py]: I'd propose to change to the following verification 
```Python
assert is_control_updateable
```

File: test/selenium/src/tests/test_snapshots.py [hunk 2]
```diff
@@ -213,22 +206,17 @@ def test_update_snapshotable_control_to_latest_ver(
...
-        src_obj=audit, obj=expected_control)
     is_control_updateable = (
         controls_ui_service.is_obj_updateble_via_info_panel(
             src_obj=audit, obj=expected_control))
+    is_control_openable = controls_ui_service.is_obj_page_exist_via_info_panel(
+        src_obj=audit, obj=expected_control)
     actual_control = controls_ui_service.get_list_objs_from_info_panels(
         src_obj=audit, objs=expected_control)
-    assert is_control_openable is is_openable
     assert is_control_updateable is False
```
Comment (dnovikau) [inline test/selenium/src/tests/test_snapshots.py]: I'd propose to change to the following verification 
```Python
assert not is_control_updateable
```

**Comment by AleksNeStu** [comment]
In order to unblock this PR I will address mentioned comments as a part of another PR. ok?

**Comment by dnovikau** [comment]
LGTM

**Pull request merged**